homeassistant/components/toon/sensor.py

```diff
@@ -49,6 +50,10 @@ async def async_setup_entry(
         )
     ]
 
+    sensors.extend(
```

Comment:
Any reason you extend here instead of adding it to the list comprehension above?